packages/wonder-blocks-modal/generated-snapshot.test.js

```diff
@@ -12,7 +12,6 @@ import ModalLauncher from "./components/modal-launcher.js";
 import OnePaneDialog from "./components/one-pane-dialog/one-pane-dialog.js";
 import ModalDialog from "./components/modal-dialog.js";
 import ModalPanel from "./components/modal-panel.js";
-import ModalContent from "./components/modal-content.js";
```

Comment:
I suspect that is b/c I removed that from `styleguide.config.js`. I noticed I didn't need to add it in the TwoPaneDialog example.

Then, It would make sense(?) if we don't expose this block, mostly because it's only used/consumed internally by `ModalPanel`. Any thoughts?